chemprop/models/model.py

```diff
@@ -62,7 +62,7 @@ def create_ffn(self, args: TrainArgs) -> None:
         if args.features_only:
             first_linear_dim = args.features_size
```

Comment:
If the use only uses features, is it still possible to use number_of_molecules > 1? If so, does this line need to be changed?